content/blog/tidymodels-2022-q2/index.Rmd

```diff
@@ -0,0 +1,206 @@
+---
+output: hugodown::hugo_document
+
+slug: tidymodels-2022-q2
+title: "Q2 2022 tidymodels digest"
+date: 2022-06-24
+author: Emil Hvitfeldt
+description: >
+    Q2 marks the end of the season of case weights, with TKTK new releases.
+
+photo:
+  url: https://unsplash.com/photos/BBR_zigEmyQ
+  author:  Samuel Girven
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [roundup] 
+tags: [tidymodels, parsnip, recipes]
+---
+
+<!--
+TODO:
+* [X] Look over / edit the post's title in the yaml
+* [X] Edit (or delete) the description; note this appears in the Twitter card
+* [X] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [X] Find photo & update yaml metadata
+* [X] Create `thumbnail-sq.jpg`; height and width should be equal
+* [X] Create `thumbnail-wd.jpg`; width should be >5x height
+* [X] `hugodown::use_tidy_thumbnails()`
+* [X] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
```

Comment:
This is being left unchecked to remind myself to rerun the functions at the end of Q2

Reply:
One other thing to be sure of in the final rerun is to have only CRAN versions of tidymodels packages installed. 👍